source/faq.md

```diff
@@ -0,0 +1,308 @@
+This means we recommend one of the following or equivalent:
+- A `.sdf`, `.mol`, or `.mol2` file or files for the molecules comprising your system, with correct bond orders and formal charges. (Note: Do NOT generate this from a simulation package or tool which does not have access to bond order information; you may end up with a correct-seeming file, but the bond orders will be incorrect)
+- Isomeric SMILES strings for the components of your system
+- InChi strings for the components of your system
+- Chemical Identity Registry numbers for the components of your system
+- IUPAC names for the components of your system
+
+Essentially, anything which provides the full identity of what you want to simulate (including stereochemistry) should work, though it may require more or less work to get it into an acceptable format.
+
+:::::
+
+:::::{faq-entry} How can I transfer my prepared system to HPC resources for simulation?
+
+OpenFF recommends exporting a prepared `Interchange` to the target MD engine and using the MD engine's recommended method to transfer it to HPC resources. This way, no additional dependencies need to be installed on the HPC resource to use OpenFF tools during preparation. For most MD engines, simply transfer the files produced by the appropriate [`Interchange.to_*()`] methods. For OpenMM, create a `System` Python object with [`Interchange.to_openmm_system()`] or [`ForceField.create_openmm_system()`] and transfer it by [serializing to XML].
+
+[`Interchange.to_*()`]: https://docs.openforcefield.org/projects/interchange/en/stable/_autosummary/openff.interchange.Interchange.html
+[`Interchange.to_openmm_system()`]: https://docs.openforcefield.org/projects/interchange/en/stable/_autosummary/openff.interchange.Interchange.html#openff.interchange.Interchange.to_openmm_system
+[`ForceField.create_openmm_system()`]: https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.typing.engines.smirnoff.ForceField.html#openff.toolkit.typing.engines.smirnoff.ForceField.create_openmm_system
+[serializing to XML]: https://openmm.github.io/openmm-cookbook/latest/notebooks/cookbook/Saving%20Systems%20to%20XML%20Files.html
+:::::
+
+## Errors and Performance Issues
+
+:::::{faq-entry} Why does partial charge assignment fail during conformer generation, even though my molecule has conformers?
+
+Assigning partial charges with a quantum chemical method requires conformers, as they are an essential input to a quantum chemical calculation.
+Because the charges assigned by a SMIRNOFF force field should be transferrable between systems, we default to generating our own set of conformers during charge assignment.
+This requirement will become unnecessary for future SMIRNOFF force fields that use NAGL graph charges; see the [](#under-the-hood) section.
+
+To assign charges based on the provided conformer if conformer generation fails, first assign charges, then use the assigned charges during parametrization:
```

Comment:
I get the message here, but the phrasing is (to me) not detailed enough on the specific point of "you can use the API to use _your_ conformer(s) if conformer generation fails." The `use_conformers` argument isn't explicitly called out in the sentences above and I think it'd be useful to call it out here, i.e.
```suggestion
To assign charges based on the provided conformer if conformer generation fails, you can use a `try:`/`except:` block and the `use_conformers` argument when assigning charges. These charges can then be used charges during parametrization with the `charge_from_molecules` argument:
```